### PR TITLE
✅ [Story interactive] Visual diff tests for image quizzes and polls

### PR DESCRIPTION
**Pull request by Brandons42**
Added Percy visual diff tests for basic image quizzes and polls (separate test for each) with default customization options

/cc @mszylkowski 
/cc @processprocess

### CODE REVIEW

File: test/visual-diff/visual-tests [hunk 1]
```diff
@@ -695,6 +695,22 @@
         ".i-amphtml-story-loaded",
       ],
     },
+    {
+      "url": "examples/visual-tests/amp-story/amp-story-interactive-img-poll.html",
+      "name": "amp-story-interactive-img-poll: basic test",
```
Comment (processprocess) [inline test/visual-diff/visual-tests:700]: Updating the name to a bit more descriptive. 
Rendering 4 images in mobile layout.

File: test/visual-diff/visual-tests [hunk 2]
```diff
@@ -695,6 +695,22 @@
...
+      "viewport": {"width": 320, "height": 480},
+      "loading_complete_selectors": [
+        ".i-amphtml-story-loaded",
+      ],
+    },
+    {
+      "url": "examples/visual-tests/amp-story/amp-story-interactive-img-quiz.html",
+      "name": "amp-story-interactive-img-quiz: basic test",
```
Comment (processprocess) [inline test/visual-diff/visual-tests:708]: Same comment as above.

**Review by processprocess — APPROVED** [review]
These visual tests will be very helpful while developing the feature and beyond.
Looks great!

**Comment by processprocess** [comment]
Friendly ping:
You'll need to add `examples/visual-tests/amp-story/amp-story-interactive-img-poll.html` to the `htmlFixtureGlobs` for [this check](https://app.circleci.com/pipelines/github/ampproject/amphtml/12051/workflows/ced00916-a44b-4b4a-9c4c-93213bba9527/jobs/191984) to pass.

**Comment by mszylkowski** [comment]
Might not be necessary to add the tests to `htmlFixtureGlobs` after #35026

**Review by mszylkowski — APPROVED** [review]

**Comment by processprocess** [comment]
> Might not be necessary to add the tests to `htmlFixtureGlobs` after #35026

Awesome! Would a `git pull upstream main`, and then pushing help this pass?

**Comment by Brandons42** [comment]
> Awesome! Would a `git pull upstream main`, and then pushing help this pass?

I'll try it out

**Pull request merged**